### PR TITLE
Added list of available games on homepage

### PR DESCRIPTION
**Pull request by eyepenn**
Added list of games with an open seat. Added a "Join game" link for open seat.

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 class GamesController < ApplicationController
+  
+  def index
+    @game = Game.where("white_player_id is NULL OR black_player_id is NULL").order('created_at').reverse_order
```
Comment (ijones16) [inline app/controllers/games_controller.rb]: This should be plural

Comment (eyepenn) [inline app/controllers/games_controller.rb]: I'm not sure what you mean. Can you post an example?

Comment (ijones16) [inline app/controllers/games_controller.rb]: I just mean that @game should be @games because it refers to many games :)

Comment (eyepenn) [inline app/controllers/games_controller.rb]: Makes sense.

**Comment by mscoutermarsh** [comment]
Looks good! Run `rubocop -a` to fix up codestyle / get the build green. 👍

**Comment by mscoutermarsh** [comment]
Everything looks good - ^ need to resolve merge conflict. Can you do that and merge it to master?

**Comment by eyepenn** [comment]
I tried to merge this. Not sure if I got it right.

**Comment by hughesbm** [comment]
Looks like you still have a merge conflict with index.html.erb

**Comment by mscoutermarsh** [comment]
Closing - was merged in other PR

**Pull request closed without merging**